src/index.js

```diff
@@ -1,8 +1,3 @@
 import SmartBanner from './smartbanner.js';
 
-let smartbanner;
-
-window.addEventListener('load', function() {
-  smartbanner = new SmartBanner();
-  smartbanner.publish();
-});
+window.SmartBanner = SmartBanner;
```

Comment:
Sure why not.
It could introduce change without breaking anything and you could remove this transition later in a new major version (if you want to)